src/components/forms/Dropdown/Dropdown.test.tsx

```diff
@@ -1,18 +1,24 @@
 import React from 'react'
 import { render } from '@testing-library/react'
+jest.mock('../../../deprecation')
+import { deprecationWarning } from '../../../deprecation'
 
-import { Select } from '../Select/Select'
+import { Dropdown } from './Dropdown'
 
 describe('Dropdown component', () => {
-  it('renders without errors', () => {
+  it('renders without errors, displaying a deprecation warning', () => {
     const { queryByTestId } = render(
-      <Select id="input-type-text" name="input-type-text">
+      <Dropdown id="input-type-text" name="input-type-text">
```

Comment:
This test should still be for the Dropdown component. Might've accidentally gotten updated by IDE in #2415